installers/rpm-code/codes/mgis.sh

```diff
@@ -2,6 +2,26 @@
 
 mgis_main() {
     codes_dependencies common boost fenics
+    mgis_mgis
+    mgis_mfront
+}
+
+mgis_mfront() {
+    codes_download thelfer/tfel TFEL-4.0.0
+    codes_cmake \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX="${codes_dir[prefix]}" \
+        -DPYTHON_INCLUDE_DIR="$(codes_python_include_dir)"\
+        -DPYTHON_LIBRARY="$(codes_python_lib_dir)" \
+        -Denable-python=ON \
+        -Denable-reference-doc=OFF \
+        -Denable-website=OFF
+    codes_make
+    codes_make_install
+}
+
+mgis_mgis() {
+    declare p="$PWD"
     codes_download thelfer/MFrontGenericInterfaceSupport
     local f
```

Comment:
local to declare throughout

Reply:
https://github.com/radiasoft/download/pull/339/commits/8cfcf3474dd01f282070a20986449709eb70a57e